src/app/query/bitcoin/address/utxos-by-address.hooks.ts

```diff
@@ -12,16 +13,12 @@ const defaultArgs = {
  * Warning: ⚠️ To avoid spending inscriptions, when using UTXOs
  * we set `filterInscriptionUtxos` and `filterPendingTxsUtxos` to true
  */
-export function useCurrentNativeSegwitUtxos(args = defaultArgs) {
-  const { filterInscriptionUtxos, filterPendingTxsUtxos, filterRunesUtxos } = args;
-
+export function useCurrentNativeSegwitUtxos() {
   const nativeSegwitSigner = useCurrentAccountNativeSegwitIndexZeroSignerNullable();
   const address = nativeSegwitSigner?.address ?? '';
+  const spendableUtxos = useInscribedSpendableUtxos();
+
+  const query = useNativeSegwitUtxosByAddress({ address, ...defaultArgs });
 
-  return useNativeSegwitUtxosByAddress({
-    address,
-    filterInscriptionUtxos,
-    filterPendingTxsUtxos,
-    filterRunesUtxos,
-  });
+  return { ...query, data: [...(query.data ?? []), ...spendableUtxos] };
```

Comment:
Maybe smth like this would be easier to understand
```suggestion
  const queryResponseData = query.data ?? []
  return { ...query, data: [...queryResponseData, ...spendableUtxos] };
```